### PR TITLE
Remove redundant suppression warnings and fixed some formatting in PoolingConnection.java

### PR DESCRIPTION
**Pull request by rajucomp**
Thanks for your contribution to [Apache Commons](https://commons.apache.org/)! Your help is appreciated!

Before you push a pull request, review this list:

- [x] Read the [contribution guidelines](CONTRIBUTING.md) for this project.
- [x] Run a successful build using the default [Maven](https://maven.apache.org/) goal with `mvn`; that's `mvn` on the command line by itself.
- [x] Write unit tests that match behavioral changes, where the tests fail if the changes to the runtime are not applied. This may not always be possible but is a best-practice.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Each commit in the pull request should have a meaningful subject line and body. Note that commits might be squashed by a maintainer on merge.

### CODE REVIEW

**Comment by rajucomp** [comment]
@garydgregory Can I get a review please ?

File: src/main/java/org/apache/commons/dbcp2/PoolingConnection.java [hunk 1]
```diff
@@ -47,7 +46,7 @@ public class PoolingConnection extends DelegatingConnection<Connection>
 
     /**
      * Statement types.
-     *
+     * <p>
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/dbcp2/PoolingConnection.java:49]: You'll want to close HTML tags.

File: src/main/java/org/apache/commons/dbcp2/PoolingConnection.java [hunk 2]
```diff
@@ -122,7 +121,6 @@ public synchronized void close() throws SQLException {
             }
         } finally {
             try {
-                @SuppressWarnings("resource")
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/dbcp2/PoolingConnection.java:125]: Do not remove `SuppressWarnings`. There are there for a reason: IDEs can be configured to show warnings, this documents that a warning can be ignored and shows when Closeable and AutoCloseable resources are intentionally left open.

**Review by garydgregory — CHANGES_REQUESTED** [review]
Hello @rajucomp 

Thank you for your PR. Please see my comments.

File: src/main/java/org/apache/commons/dbcp2/PoolingConnection.java (hunk 2 above)
Comment (rajucomp) [reply src/main/java/org/apache/commons/dbcp2/PoolingConnection.java:125]: I think modern versions of IntelliJ suggest to remove this warnings. Thanks for the context though. With that being said, there is no useful changes in the PR to merit a review. I am gonna close this. Sorry for wasting your time.

**Pull request closed without merging**